fuel-coroutines/src/main/kotlin/com/github/kittinunf/fuel/coroutines/Coroutines.kt

```diff
@@ -118,11 +118,3 @@ suspend fun <U : Any> Request.awaitObjectResult(
     }
     Result.Failure(fuelError)
 }
-
-@Deprecated("please use 'awaitByteArray()'", ReplaceWith("awaitByteArray()", "deserializable"))
-suspend fun Request.awaitResponseResult(): ByteArray = awaitByteArray()
-
-@Deprecated("please use 'awaitObjectResult(deserializable)'", ReplaceWith("awaitObjectResult(deserializable)"))
-suspend fun <U : Any> Request.awaitSafelyObjectResult(
-    deserializable: ResponseDeserializable<U>
```

Comment:
do we have a policy on how long we leave deprecated methods in the lib before we remove them?

Reply:
I tend to keep them around at least 2 major versions (so since 1.14), but just take the opportunity to remove them as we don't test them anymore